konnectd/ui/build/static/js/10.fe240f90.chunk.js

```diff
@@ -0,0 +1,3 @@
a.Fragment,null,s.a.cloneElement(i,Object(o.a)({ref:Ce},Fe)),s.a.createElement(E,Object(o.a)({className:Object(d.a)(c.popper,L&&c.popperInteractive,a&&c.popperArrow),placement:A,anchorEl:X,open:!!X&&de,id:Fe["aria-describedby"],transition:!0,popperOptions:{modifiers:{arrow:{enabled:Boolean(Z),element:Z}}}},Le,q),(function(e){var t=e.placement,r=e.TransitionProps;return s.a.createElement(U,Object(o.a)({timeout:G.transitions.duration.shorter},r,Y),s.a.createElement("div",{className:Object(d.a)(c.tooltip,c["tooltipPlacement".concat(Object(f.a)(t.split("-")[0]))],te.current&&c.touch,a&&c.tooltipArrow)},$,a?s.a.createElement("span",{className:c.arrow,ref:ee}):null))})))}));t.a=Object(p.a)((function(e){return{popper:{zIndex:e.zIndex.tooltip,pointerEvents:"none",flip:!1},popperInteractive:{pointerEvents:"auto"},popperArrow:{'&[x-placement*="bottom"] $arrow':{flip:!1,top:0,left:0,marginTop:"-0.95em",marginLeft:4,marginRight:4,width:"2em",height:"1em","&::before":{flip:!1,borderWidth:"0 1em 1em 1em",borderColor:"transparent transparent currentcolor transparent"}},'&[x-placement*="top"] $arrow':{flip:!1,bottom:0,left:0,marginBottom:"-0.95em",marginLeft:4,marginRight:4,width:"2em",height:"1em","&::before":{flip:!1,borderWidth:"1em 1em 0 1em",borderColor:"currentcolor transparent transparent transparent"}},'&[x-placement*="right"] $arrow':{flip:!1,left:0,marginLeft:"-0.95em",marginTop:4,marginBottom:4,height:"2em",width:"1em","&::before":{flip:!1,borderWidth:"1em 1em 1em 0",borderColor:"transparent currentcolor transparent transparent"}},'&[x-placement*="left"] $arrow':{flip:!1,right:0,marginRight:"-0.95em",marginTop:4,marginBottom:4,height:"2em",width:"1em","&::before":{flip:!1,borderWidth:"1em 0 1em 1em",borderColor:"transparent transparent transparent currentcolor"}}},tooltip:{backgroundColor:Object(u.b)(e.palette.grey[700],.9),borderRadius:e.shape.borderRadius,color:e.palette.common.white,fontFamily:e.typography.fontFamily,padding:"4px 8px",fontSize:e.typography.pxToRem(10),lineHeight:"".concat(S(1.4),"em"),maxWidth:300,wordWrap:"break-word",fontWeight:e.typography.fontWeightMedium},tooltipArrow:{position:"relative",margin:"0"},arrow:{position:"absolute",fontSize:6,color:Object(u.b)(e.palette.grey[700],.9),"&::before":{content:'""',margin:"auto",display:"block",width:0,height:0,borderStyle:"solid"}},touch:{padding:"8px 16px",fontSize:e.typography.pxToRem(14),lineHeight:"".concat(S(16/14),"em"),fontWeight:e.typography.fontWeightRegular},tooltipPlacementLeft:Object(a.a)({transformOrigin:"right center",margin:"0 24px "},e.breakpoints.up("sm"),{margin:"0 14px"}),tooltipPlacementRight:Object(a.a)({transformOrigin:"left center",margin:"0 24px"},e.breakpoints.up("sm"),{margin:"0 14px"}),tooltipPlacementTop:Object(a.a)({transformOrigin:"center bottom",margin:"24px 0"},e.breakpoints.up("sm"),{margin:"14px 0"}),tooltipPlacementBottom:Object(a.a)({transformOrigin:"center top",margin:"24px 0"},e.breakpoints.up("sm"),{margin:"14px 0"})}}),{name:"MuiTooltip"})(M)}}]);
```

Comment:
Codacy found an issue: [The numeric literal '.9' will have at different value at runtime.](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=6337760)